atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraVerifier.java

```diff
@@ -385,8 +388,16 @@ private static void checkRfsMatchConfig(
             Set<String> dcs,
```

Comment:
Should we refactor to unify these into a small struct?

Reply:
Do you mean the expected DCs and RF? Yeah, seems reasonable.